src/Module.ts

```diff
@@ -116,7 +116,7 @@ export interface AstContext {
 }
 
 export const defaultAcornOptions: acorn.Options = {
-	ecmaVersion: 2020,
+	ecmaVersion: 2021 as 2020,
```

Comment:
I thought optional chaining was an ES 2020 feature?

Reply:
It was implemented as 2021 in the PR there, which I'm following - I think you are right though in that it is strictly 2020.